assets/enums/singlegenome_species.json

```
@@ -37,6 +37,7 @@
         "Methanobrevibacter woesei",
         "Escherichia coli",
         "African cassava mosaic virus",
-        "Haemophilus influenzae"
+        "Haemophilus influenzae",
+        "Human herpes simplex virus 1"
```

```suggestion
        "Human alphaherpesvirus 1"
```

The name needs to follow the NCBI taxonomy: https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=10298 and https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=36345